### ISSUE TITLE
403 Forbidden errors from Twitter

### ISSUE THREAD

**Issue by OverlyDev** [OverlyDev/go-bungie-alerter#35]
**Describe the bug**
After some amount of runtime, Twitter will start responding with `403 Forbidden` errors.

**To Reproduce**
1. Run the application in debug mode using the `--debug/-d` flag
2. Observe the logs showing successful Twitter responses
3. Wait a while (maybe an hour? maybe longer?)
4. Observe `403 Forbidden` errors from Twitter

**Expected behavior**
There shouldn't be any error responses from Twitter.

**Screenshots**
![image](https://user-images.githubusercontent.com/55297573/229936971-c53c1b28-bf88-4b4d-bef4-322254ad9120.png)

**Environment:**
- BungieAlerter version: v0.1.0
 - Platform: Docker
 - Extra: N/A

**Additional context**
Some questions:
1. Exactly how long until we start receiving `403 Forbidden`?
2. Do we just need to re-auth every hour or something?
3. We shouldn't be getting rate-limited since we're under their quota

**Comment by OverlyDev** [comment]
Restarting the application returns normal functionality. Since doing this would request a new token, I'm assuming if we were to manually re-auth every 30min/1hr it would be fine.

That or just trigger a re-auth on the first `403 Forbidden` received, which would probably be the smarter approach.

**Comment by OverlyDev** [comment]
This was a fun one to test since the issue only surfaced after ~3hrs of runtime.
![image](https://user-images.githubusercontent.com/55297573/229995972-dc8d92f3-074b-4c6e-840f-f6d55d925cd5.png)

I've got logic in place to renew the `guest-token` after receiving a `403 Forbidden`.  After renewing the token, we got a successful response containing the data we need so I call that a win.

I also had it bail afterwards that way I could actually see that it did the thing, instead of having to scroll through the logs trying to find where it happened.

Should be able to get this fix pushed up tomorrow!
